FileAPI/url/url-basics.any.js

```diff
@@ -0,0 +1,63 @@
+const blob = new Blob(['test']);
+const file = new File(['test'], 'name');
+
+test(() => {
+  const url_count = 10000;
+  let list = [];
+
+  for (let i = 0; i <= url_count; ++i)
+    list.push(URL.createObjectURL(blob));
+
+  list.sort();
+
+  for (let i = 0; i < url_count; ++i)
```

Comment:
> nit 1: url_count -> list.length makes the loop slightly more self-contained

note that list.length is url_count+1, not url_count



> nit 2: You could start with i = 1 and compare list[i] against list[i - 1] to avoid accessing the element past the array's end. In this case, the out-of-bounds access doesn't cause problems because you get undefined and it is compared against a string... but I don't think a reader should have to reason through this.

which also means that this isn't a concern, we're not doing any out-of-bounds access.

But yeah, it's a good point that the difference between <= and < in the two loops is kind of subtle, and just having < in both and starting the second loop at 1 would be more intuitive. I just copied what the existing test did....